src/extension/background-script/actions/blocklist/delete.ts

```diff
@@ -0,0 +1,20 @@
+import db from "../../db";
+
+const deleteBlocklist = async (
+  message: { args: { host: string } },
+  sender: unknown
```

Comment:
If the sender isn't being used it can be removed here and for other occurrences.